src/cloudant/client.py

```diff
@@ -31,7 +31,8 @@
     USER_AGENT,
     append_response_error_content,
     InfiniteSession,
-    ClientSession)
+    ClientSession,
```

Comment:
Copyright date on this file is a bit weird, but I think it's fixed in #306 anyway.